### PR TITLE
Australia (House of Representatives): refresh data

### PR DESCRIPTION
**Pull request by everypoliticianbot**
Automated data refresh for Australia - House of Representatives

#### Output

```
Add memberships from sources/morph/openaustralia.csv
Merging with sources/morph/data.csv
Merging with sources/morph/wikidata.csv
* 33 of 508 unmatched
	{:id=>"Q25753992", :name=>"Susan Templeman"}
	{:id=>"Q25324887", :name=>"Susan Lamb"}
	{:id=>"Q24191457", :name=>"Julian Leeser"}
	{:id=>"Q20631311", :name=>"Anne Aly"}
	{:id=>"Q25352543", :name=>"Ben Morton"}
	{:id=>"Q25844494", :name=>"Emma McBride"}
	{:id=>"Q25352606", :name=>"Ross Hart"}
	{:id=>"Q25350560", :name=>"Justine Keay"}
	{:id=>"Q25752522", :name=>"Matt Keogh"}
	{:id=>"Q26695681", :name=>"Brian Mitchell"}
Merging with sources/manual/emails.csv
Merging with sources/manual/guessed-emails.csv
Adding GenderBalance results from sources/gender-balance/results.csv

GenderBalance Mismatches
    ☁ Mismatch for f24b325e-6885-4028-b414-3b98d8b287ff Ricky Johnston (Was: female | GB: male)
    ☁ Mismatch for 0448f62d-f3ab-4aa1-86da-70c3fe73912d Kym Richardson (Was: male | GB: female)
    ☁ Mismatch for 56477745-9d1a-437d-944a-95f0d8ba164a Clair St (Was: male | GB: female)

  ⚥ data for 1653; 0 added

Adding OCD areas from sources/ocd/divisions.csv
  Matched Area Northern Territory to Northern Territory

Unmatched areas
  No area match for Gwydir
  No area match for Bonython
  No area match for Prospect
  No area match for Corinella
  No area match for Kalgoorlie
  No area match for St George
  No area match for Lowe
  No area match for Namadgi
  No area match for Hawker
  No area match for Riverina-Darling
  No area match for Streeton
  No area match for Phillip
  No area match for Burke
  No area match for Dundas
  No area match for Henty
  No area match for Whitlam
  No area match for Fenner
  No area match for Burt

Applying local corrections from sources/manual/corrections.csv
No dates for election Q26271553 (Next Australian federal election)
Creating 11 term files

Top identifiers:
  512 x openaustralia
  475 x wikidata
  195 x aph
  154 x viaf
  71 x freebase

Creating names.csv
Creating unstable/positions.csv
  Unknown position (x1): Q30185 mayor — e.g. Q982227
Persons matched to Wikidata: 475 ✓ | 37 ✘
  No wikidata: Milton Dick (60108137-80b8-49bb-82e1-a96f56d1867a)
  No wikidata: Anne Stanley (d3a05e9d-7564-4c4c-81b6-2bb3c1ae48e8)
  No wikidata: Ross Hart (040f153a-ddbc-4f9e-a3f8-95fd0a97335f)
  No wikidata: Susan Lamb (83f0cfa9-ddbe-41ea-b5fa-56fad2d3eca6)
  No wikidata: Tim Hammond (a2d03383-be31-4a59-b7f4-9d178b155c52)
  No wikidata: Julia Banks (3ccc734f-f075-48c2-b2d7-4c8d6044506b)
  No wikidata: Anne Aly (71d0af02-8bd3-4dd5-8d06-0bfae6f8a5dc)
  No wikidata: Rebekha Sharkie (c0ce0d23-dea3-41e5-b4f8-fe2e9bef2bc3)
  No wikidata: Chris Crewther (414c956b-52aa-43fd-87bf-665e6c8e34da)
  No wikidata: Julian Leeser (06e71139-88f5-4b81-9f66-dea296d9daa3)
Parties matched to Wikidata: 10 ✓ | 3 ✘
  No wikidata: Liberal National Party (party/liberal_national_party)
  No wikidata: CWM (party/cwm)
  No wikidata: Nick Xenophon Team (party/nick_xenophon_team)

```

### CODE REVIEW

**Comment by ondenman** [comment]
screen shot 2016-09-15 at 12 03 19
screen shot 2016-09-15 at 12 05 36

**Comment by ondenman** [comment]
Closes: #16621

**Comment by tmtmtmtm** [comment]
Summary of changes in `data/Australia/Representatives/ep-popolo-v1.0.json`:

## People

### Added



- `eccaad3d-c3d0-4c69-91fe-881ea20a069d` - Andrew Gee

- `16521ab5-bf39-413b-afc5-7f89a14b550f` - Andrew Wallace

- `71d0af02-8bd3-4dd5-8d06-0bfae6f8a5dc` - Anne Aly

- `d3a05e9d-7564-4c4c-81b6-2bb3c1ae48e8` - Anne Stanley

- `72eed412-02a5-46ca-8321-28779fd12dbf` - Ben Morton

- `63666af9-457a-4ebc-8e95-0985d7975c62` - Brian Mitchell

- `4845aa63-ac65-473e-b8ca-dd1384f630a0` - Cathy O'Toole

- `414c956b-52aa-43fd-87bf-665e6c8e34da` - Chris Crewther

- `638a64b8-0c8d-4369-adf4-f6b8869f4156` - Damian Drum

- `36a29a52-e860-470b-bd4c-44384d85cc21` - David Littleproud

- `47743001-1ae2-49ff-8ba9-1fed876f372c` - Emma Husar

- `4307bcd9-84fb-4007-9b66-9f450c031bac` - Emma McBride

- `14446abe-9e8c-419b-adb4-a768ae52a342` - Jason Falinski

- `7564fafa-aace-4662-94a2-39d853e947ea` - John McVeigh

- `d75918bf-408e-4d8e-a309-00f6316a92c7` - Josh Wilson

- `3ccc734f-f075-48c2-b2d7-4c8d6044506b` - Julia Banks

- `5459d690-51c6-49f0-bb88-f095926bdd72` - Julian Hill

- `06e71139-88f5-4b81-9f66-dea296d9daa3` - Julian Leeser

- `d82e379c-3b63-42f2-891d-fc2b517a6d1d` - Justine Keay

- `cf3ef83d-cb1e-498e-b3f8-04c84312e5ae` - Linda Burney

- `66171983-750b-4bf5-942b-8872af80a2ba` - Llew O'Brien

- `f827cee5-c604-4a0b-93ca-58ec675ec54b` - Luke Gosling

- `e1d395c9-08a2-47f3-847c-b6d54681a3a9` - Madeleine King

- `55b18048-2132-4c96-899e-993277fcd2e9` - Matt Keogh

- `4fe8021f-12ba-4f12-ae45-e7f49514d842` - Meryl Swanson

- `fd0e4413-2bd4-41ef-9a73-35aa96392463` - Mike Freelander

- `60108137-80b8-49bb-82e1-a96f56d1867a` - Milton Dick

- `1329beeb-2b7b-4223-b31e-cc2c8e3a92bc` - Nicolle Flint

- `2dff2dde-a1e2-471b-ae84-a798f6546667` - Peter Khalil

- `c0ce0d23-dea3-41e5-b4f8-fe2e9bef2bc3` - Rebekha Sharkie

- `040f153a-ddbc-4f9e-a3f8-95fd0a97335f` - Ross Hart

- `83f0cfa9-ddbe-41ea-b5fa-56fad2d3eca6` - Susan Lamb

- `923bd346-2669-4241-8a33-b495d2388cb4` - Susan Templeman

- `ff2cfb12-871c-4bf7-ae6a-9fd2c8262978` - Ted O'Brien

- `a2d03383-be31-4a59-b7f4-9d178b155c52` - Tim Hammond

- `cfd367d9-6bb6-452a-a9f3-9c8adeeb344d` - Tim Wilson

- `0e1881a5-8c9e-43ce-b491-6b33c236dfcf` - Trevor Evans



### Removed


No people removed


### Name Changes


No name changes


### Additional Name Changes


No name changes


### Wikidata Changes


No changes


## Organizations

### Added



- `party/liberal_national_party` - Liberal National Party

- `party/nick_xenophon_team` - Nick Xenophon Team



### Removed


No organizations removed


## Memberships

### Added


#### term/44

- Bob Baldwin ( - 2016-05-09)

- Bruce Billson ( - 2016-05-09)

- Bronwyn Bishop (2015-08-02 - 2016-05-09)

- Mal Brough ( - 2016-05-09)

- Anna Burke ( - 2016-05-09)

- John Cobb ( - 2016-05-09)

- Laurie Ferguson ( - 2016-05-09)

- Teresa Gambaro ( - 2016-05-09)

- Gary Gray ( - 2016-05-09)

- Alan Griffin ( - 2016-05-09)

- Jill Hall ( - 2016-05-09)

- Dennis Jensen ( - 2016-05-09)

- Ian Macfarlane ( - 2016-05-09)

- Alannah Mactiernan ( - 2016-05-09)

- Clive Palmer ( - 2016-05-09)

- Melissa Parke ( - 2016-05-09)

- Bernie Ripoll ( - 2016-05-09)

- Andrew Robb ( - 2016-05-09)

- Philip Ruddock ( - 2016-05-09)

- Bruce Scott ( - 2016-05-09)

- Andrew Southcott ( - 2016-05-09)

- Sharman Stone ( - 2016-05-09)

- Kelvin Thomson ( - 2016-05-09)

- Warren Truss ( - 2016-05-09)


#### term/45

- Tony Abbott 

- Anthony Albanese 

- John Alexander 

- Anne Aly (2016-07-02 - )

- Karen Andrews 

- Kevin Andrews 

- Bob Baldwin ( - 2016-05-09)

- Adam Bandt 

- Julia Banks (2016-07-02 - )

- Bruce Billson ( - 2016-05-09)

- Sharon Bird 

- Bronwyn Bishop ( - 2016-05-09)

- Julie Bishop 

- Chris Bowen 

- Jamie Briggs ( - 2016-07-02)

- Andrew Broad 

- Russell Broadbent 

- Gai Brodtmann 

- Mal Brough ( - 2016-05-09)

- Scott Buchholz 

- Anna Burke ( - 2016-05-09)

- Tony Burke 

- Linda Burney (2016-07-02 - )

- Mark Butler 

- Terri Butler 

- Anthony Byrne 

- Jim Chalmers 

- Nick Champion 

- Darren Chester 

- Lisa Chesters 

- George Christensen 

- Steven Ciobo 

- Jason Clare 

- Sharon Claydon 

- John Cobb ( - 2016-05-09)

- David Coleman 

- Julie Collins 

- Pat Conroy (2016-07-02 - )

- Pat Conroy ( - 2016-07-02)

- Mark Coulton ( - 2016-08-30)

- Mark Coulton (2016-08-30 - )

- Chris Crewther (2016-07-02 - )

- Michael Danby 

- Milton Dick (2016-07-02 - )

- Mark Dreyfus 

- Damian Drum (2016-07-02 - )

- Peter Dutton 

- Justine Elliot 

- Kate Ellis 

- Warren Entsch 

- Trevor Evans (2016-07-02 - )

- Jason Falinski (2016-07-02 - )

- David Feeney 

- Laurie Ferguson ( - 2016-05-09)

- Joel Fitzgibbon 

- Paul Fletcher 

- Nicolle Flint (2016-07-02 - )

- Mike Freelander (2016-07-02 - )

- Josh Frydenberg 

- Teresa Gambaro ( - 2016-05-09)

- Andrew Gee (2016-07-02 - )

- Steve Georganas (2016-07-02 - )

- Andrew Giles 

- David Gillespie 

- Ian Goodenough 

- Luke Gosling (2016-07-02 - )

- Gary Gray ( - 2016-05-09)

- Alan Griffin ( - 2016-05-09)

- Natasha Griggs ( - 2016-07-02)

- Jill Hall ( - 2016-05-09)

- Tim Hammond (2016-07-02 - )

- Ross Hart (2016-07-02 - )

- Luke Hartsuyker 

- Andrew Hastie 

- Alex Hawke 

- Chris Hayes 

- Sarah Henderson 

- Peter Hendy ( - 2016-07-02)

- Julian Hill (2016-07-02 - )

- Kevin Hogan 

- Luke Howarth 

- Greg Hunt 

- Emma Husar (2016-07-02 - )

- Ed Husic 

- Eric Hutchinson ( - 2016-07-02)

- Steve Irons 

- Dennis Jensen ( - 2016-05-09)

- Ewen Jones ( - 2016-07-02)

- Stephen Jones ( - 2016-07-02)

- Stephen Jones (2016-07-02 - )

- Barnaby Joyce 

- Bob Katter 

- Justine Keay (2016-07-02 - )

- Michael Keenan 

- Craig Kelly 

- Mike Kelly (2016-07-02 - )

- Matt Keogh (2016-07-02 - )

- Peter Khalil (2016-07-02 - )

- Catherine King 

- Madeleine King (2016-07-02 - )

- Susan Lamb (2016-07-02 - )

- Andrew Laming 

- Michelle Landry 

- Craig Laundy 

- Julian Leeser (2016-07-02 - )

- Andrew Leigh ( - 2016-07-02)

- Andrew Leigh (2016-07-02 - )

- Sussan Ley 

- David Littleproud (2016-07-02 - )

- Ian Macfarlane ( - 2016-05-09)

- Jenny Macklin 

- Alannah Mactiernan ( - 2016-05-09)

- Nola Marino 

- Louise Markus ( - 2016-07-02)

- Richard Marles 

- Russell Matheson ( - 2016-07-02)

- Emma McBride (2016-07-02 - )

- Michael McCormack 

- Cathy McGowan 

- Karen McNamara ( - 2016-07-02)

- John McVeigh (2016-07-02 - )

- Brian Mitchell (2016-07-02 - )

- Rob Mitchell 

- Scott Morrison 

- Ben Morton (2016-07-02 - )

- Shayne Neumann 

- Andrew Nikolic ( - 2016-07-02)

- Llew O'Brien (2016-07-02 - )

- Ted O'Brien (2016-07-02 - )

- Brendan O'Connor 

- Ken O'Dowd 

- Kelly O'Dwyer 

- Clare O'Neil 

- Cathy O'Toole (2016-07-02 - )

- Julie Owens 

- Clive Palmer ( - 2016-05-09)

- Melissa Parke ( - 2016-05-09)

- Tony Pasin 

- Graham Perrett 

- Keith Pitt 

- Tanya Plibersek 

- Christian Porter 

- Jane Prentice 

- Melissa Price 

- Christopher Pyne 

- Rowan Ramsey 

- Bernie Ripoll ( - 2016-05-09)

- Amanda Rishworth 

- Andrew Robb ( - 2016-05-09)

- Stuart Robert 

- Michelle Rowland 

- Wyatt Roy ( - 2016-07-02)

- Philip Ruddock ( - 2016-05-09)

- Joanne Ryan 

- Bruce Scott ( - 2016-05-09)

- Fiona Scott ( - 2016-07-02)

- Rebekha Sharkie (2016-07-02 - )

- Bill Shorten 

- Luke Simpkins ( - 2016-07-02)

- Tony Smith 

- Warren Snowdon 

- Andrew Southcott ( - 2016-05-09)

- Anne Stanley (2016-07-02 - )

- Sharman Stone ( - 2016-05-09)

- Ann Sudmalis 

- Michael Sukkar 

- Wayne Swan 

- Meryl Swanson (2016-07-02 - )

- Angus Taylor 

- Dan Tehan 

- Susan Templeman (2016-07-02 - )

- Matt Thistlethwaite 

- Kelvin Thomson ( - 2016-05-09)

- Warren Truss ( - 2016-05-09)

- Alan Tudge 

- Malcolm Turnbull 

- Maria Vamvakinou 

- Bert Van Manen 

- Nickolas Varvaris ( - 2016-07-02)

- Ross Vasta 

- Andrew Wallace (2016-07-02 - )

- Tim Watts 

- Brett Whiteley ( - 2016-07-02)

- Lucy Wicks 

- Andrew Wilkie 

- Matt Williams ( - 2016-07-02)

- Josh Wilson (2016-07-02 - )

- Rick Wilson 

- Tim Wilson (2016-07-02 - )

- Jason Wood 

- Ken Wyatt 

- Tony Zappia 

- Trent Zimmerman 




### Removed


#### term/44

- Bob Baldwin 

- Bruce Billson 

- Bronwyn Bishop (2015-08-02 - )

- Mal Brough 

- Anna Burke 

- John Cobb 

- Laurie Ferguson 

- Teresa Gambaro 

- Gary Gray 

- Alan Griffin 

- Jill Hall 

- Dennis Jensen 

- Ian Macfarlane 

- Alannah Mactiernan 

- Clive Palmer 

- Melissa Parke 

- Bernie Ripoll 

- Andrew Robb 

- Philip Ruddock 

- Bruce Scott 

- Andrew Southcott 

- Sharman Stone 

- Kelvin Thomson 

- Warren Truss 




## Terms

### Added



- `term/45` - 45th Parliament



### Removed


No terms removed


## Elections

### Added


No elections added


### Removed


No elections removed

File: data/Australia/Representatives/sources/morph/terms.csv [hunk 1]
```diff
@@ -8,4 +8,5 @@ id,name,start_date,end_date
 41,41st Parliament,2004-10-09,2007-11-24
 42,42nd Parliament,2007-11-24,2010-08-21
 43,43rd Parliament,2010-08-21,2013-09-07
-44,44th Parliament,2013-09-07,
+44,44th Parliament,2013-09-07,2016-06-05
+45,45th Parliament,2016-02-07,
```
Comment (tmtmtmtm) [inline data/Australia/Representatives/sources/morph/terms.csv]: Ah, this explains some of the problem. The 45th Parliament is listed here as starting 4 months before the 44th Parliament ended!

**Comment by tmtmtmtm** [comment]
superseded

**Pull request closed without merging**